charts/tidb-operator/values.yaml

```diff
@@ -185,30 +185,27 @@ admissionWebhook:
   ## if admissionWebhook.create and admissionWebhook.hooksEnabled.pods are both enabled,
   ## The pre-delete-job would delete the validationWebhookConfiguration using this image
   jobImage: "bitnami/kubectl:latest"
-  hooksEnabled:
```

Comment:
hooksEnabled is confusing, remove it and use `validation` and `mutation` instead.